guides/release/contributing/index.md

```diff
@@ -23,6 +23,18 @@ Have a look at the [list of main repositories](repositories) to learn about the
 
 If you need some inspiration, you can check out the [Help Wanted](https://help-wanted.emberjs.com/) dashboard to browse for issues. If you are a beginner, look out for issues with the "Help wanted" and "Good first issue" labels.
 
+# Changing the API documentation
```

Comment:
Let's use the word "update," which may sound more positive and refer to contributing to an open-source.

```suggestion
# Updating the API documentation
```

The use of `#` will create an `<h1>` tag (and `##` an `<h2>` tag, and so on). I noticed this Markdown file and 2 other files in `/release/contributing` folder misused heading levels.

In a separate pull request, would you like to help with fixing the heading levels?

Reply:
@ijlee2 Happy to help with separate PR/issue for the headings.